content/cucumber/tags.md

```diff
@@ -26,27 +24,31 @@ A Scenario or Feature can have as many Tags as you like. Just separate them with
 Feature: Verify billing
```

Comment:
This renders funny. It a mistake or a feature?

Reply:
Can you be more specific?

Reply:
@mlvandijk took a screenshot and shared it in slack. I share here image here,
![screenshot from 2017-11-09 12-37-02](https://user-images.githubusercontent.com/541840/32603638-bbfe516e-c54a-11e7-87cd-d14df28130cb.png)

It might be fixed.

Reply:
Still a problem with text marked with 'shell':
<img width="905" alt="screen shot 2017-11-09 at 12 48 11" src="https://user-images.githubusercontent.com/20927578/32604061-7215b7f2-c54c-11e7-9756-0a5b56241f3d.png">
If possible please fix after merging the text changes, to prevent merge conflicts